frontend/public/components/cluster-settings/cluster-settings.tsx

```diff
@@ -582,43 +623,69 @@ export const NodesUpdatesGroup: React.FC<NodesUpdatesGroupProps> = ({
   const MCPUpdatingTime = getUpdatingTimeForMCP(machineConfigPool);
   const totalMCPNodes = machineConfigPool?.status?.machineCount || 0;
   const updatedMCPNodes =
-    updatedMachineCountReady || MCPUpdatingTime > updateStartedTime
+    updatedMachineCountReady || (MCPUpdatingTime > updateStartedTime && renderedConfigIsUpdated)
       ? machineConfigPool?.status?.updatedMachineCount
       : 0;
   const percentMCPNodes = calculatePercentage(updatedMCPNodes, totalMCPNodes);
+  const isUpdated = percentMCPNodes === 100;
   const { t } = useTranslation();
-  return hideIfComplete && percentMCPNodes === 100
+  return hideIfComplete && isUpdated
     ? null
-    : machineConfigOperatorLoaded && (
+    : machineConfigOperatorLoaded && renderedConfigLoaded && (
         <UpdatesGroup divided={divided}>
           <UpdatesType>
-            <Link to={`/k8s/cluster/nodes?rowFilter-node-role=${machineConfigPool.metadata.name}`}>
-              {t('public~{{name}} Nodes', { name })}
+            <Link
+              to={`/k8s/cluster/nodes?rowFilter-node-role=${machineConfigPool?.metadata?.name}`}
+            >
+              {`${name} ${NodeModel.labelPlural}`}
             </Link>
-            {name !== 'Master' && (
+            {!isMaster && (
               <FieldLevelHelp>
                 {t(
-                  'public~{{name}} {{resource}} may continue to update after the update of Master {{resource}} and {{resource2}} are complete.',
+                  'public~{{name}} {{resource}} may continue to update after the update of {{master}} {{resource}} and {{resource2}} are complete.',
                   {
                     name,
                     resource: NodeModel.labelPlural,
+                    master: NodeTypeNames.Master,
                     resource2: ClusterOperatorModel.labelPlural,
                   },
                 )}
               </FieldLevelHelp>
             )}
           </UpdatesType>
           <UpdatesBar>
-            <Progress
-              title={t('public~{{updatedMCPNodes}} of {{totalMCPNodes}}', {
-                updatedMCPNodes,
-                totalMCPNodes,
-              })}
-              value={!_.isNaN(percentMCPNodes) ? percentMCPNodes : null}
-              size={ProgressSize.sm}
-              variant={percentMCPNodes === 100 ? ProgressVariant.success : null}
-            />
+            {!isMaster && isPaused && !isUpdated ? (
+              <p>
+                <PauseCircleIcon />{' '}
+                {/* TODO:  change second sentence to 'Must resume before MM/DD/YY.'
+                once https://issues.redhat.com/browse/MCO-77 is complete */}
```

Comment:
Do we have a client-side ticket to handle this? MCO-77 is an Epic so do we want to create a story on our side / on their side to adjust this to have the right value?

Reply:
MCO-77 was the issue I was referred to by @jkyros when we discussed the need for the annotation.  Not sure how he is tracking the work on his side.  See https://issues.redhat.com/browse/MCO-77?focusedCommentId=19968846&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-19968846

Reply:
Right... but unless you're holding your PR up for this backend change, we need a ticket to manage this UI change. So we should create something so it does not get lost. If you're unsure where to create it, I suggest the CONSOLE project. Maybe on the parent epic? https://issues.redhat.com/browse/CONSOLE-2927

Doesn't hurt to create a placeholder ticket so we don't lose this in the cracks. If you are waiting for it before we merge, perhaps put a `/hold` on your PR with that as a note.

Reply:
Good point.  Added [CONSOLE-3126](https://issues.redhat.com/browse/CONSOLE-3126).